### DISCUSSION TITLE
Help Getting HAOS Notifications with Images Working

### DISCUSSION THREAD

**Discussion by kylegordy** [blakeblackshear/frigate#1785] [Ask A Question] [answered]
Hey everyone :wave: I got regular push notifications with a title and message working great. However, I'm having a hard time figuring out how to incorporate an image preview into the mix. I'm new to both Frigate and Home Assistant, so I'm sure I'm doing something wrong.

I've read over the docs [here](https://blakeblackshear.github.io/frigate/usage/home-assistant#api) and can't get any of these examples with images to work.

I'm running Home Assistant OS on a Raspberry Pi. I have the Frigate add-on installed and working fine. I also have HACS setup with the custom Frigate integration. I don't have any external public URLs configured. I do have Tailscale working to view HA outside my network.

The error I keep getting is:
`Error: Error rendering data template: UndefinedError: 'dict object' has no attribute 'payload_json'`

I'm pretty sure the API URL I'm using isn't correct, but I'm unsure what the correct one would be. 

My HA URL inside my network is `http://192.168.1.220:8123/` the URL I use to view/use the Frigate UI internally is `http://192.168.1.220:8123/ccab4aaf_frigate/`

I'm probably looking over something silly. Any help would be appreciated. I love the project!

**Comment by rhatguy** [comment, marked as answer]
Have you tried the home assistant/frigate blueprint.  I suspect it does exactly what you want to do.

https://community.home-assistant.io/t/frigate-mobile-app-notifications/311091

**Reply by kylegordy** [reply]
@rhatguy I have not, this totally worked. Thanks so much!

**Comment by haggercody** [comment]
i like the telegram method (explained in the docs) because i can keep the feed with the image and look back to it.

**Comment by Minglarn** [comment]
Or you could use Pushover... Similar to Telegram.. 
There is a feed where all the images etc are viewable ...
